app/components/key_pair_generator_component.rb

```diff
@@ -0,0 +1,11 @@
+class KeyPairGeneratorComponent < BaseComponent
+  attr_reader :location
+
+  def initialize(location:)
+    @location = location
+  end
+
+  def render?
+    AbTests::KEY_PAIR_GENERATION.bucket(SecureRandom.uuid) == :key_pair_group
```

Comment:
wait this is just completely random? I guess we don't know which user at this point